builder/table_builder.go

```diff
@@ -1,15 +1,29 @@
-package build
+package builder
 
 import (
 	"GopherDB/types"
+	"context"
 	"database/sql"
+	"fmt"
 	"log"
 	"strings"
 )
 
 func BuildTable(db *sql.DB) ([]string, []string) {
-	rows, err := db.Query(`SELECT * FROM "public"."` + strings.ToLower(types.TableSelection) + `" LIMIT 300 OFFSET 0;`)
-	defer rows.Close()
+	tableName := fmt.Sprintf("public.%s", strings.ToLower(types.TableSelection))
+	const (
+		maxRows = 300
+		offset  = 0
+	)
+	query := fmt.Sprintf("SELECT * FROM %s LIMIT $1 OFFSET $2;", tableName)
```

Comment:
Separating the query template and the actual parameters makes it easier to understand the code, and is also less error-prone in terms of changing it in the future.